### ISSUE TITLE
Add OSC support

### ISSUE THREAD

**Issue by jamiebullock** [BirminghamConservatoire/IntegraLive#961]
Integra Live needs OSC support.

I'm not sure how this should work for now...

**Comment by elmiquito** [comment]
Has there been any progress on this? Is it still on the roadmap? Seems an important feature for interoperability with other audio/video control and  processing software

**Comment by jamiebullock** [comment]
@elmiquito It's still on the roadmap but it's not really high priority since control-level interoperability is already provided through MIDI. 

Most of the relevant audio and video applications (including those on iOS) support MIDI I/O. These show up in the IL Prefs as shown below (Max and VDMX listed). The number of apps and platforms that support MIDI greatly exceeds those that support OSC and since it was released nearly 20 years ago OSC hasn't been widely adopted outside niche groups.

For me higher priorities would therefore be providing better MIDI output support for sending MID from IL (so we can control other applications from IL) and MIDI clock support, so timelines can be synced. This would allow the Ableton Live timeline to sync with that of IL, for example. I also think we should consider [ReWire](https://en.wikipedia.org/wiki/ReWire) support at some point.

The reason I still think we need OSC is for increased resolution for things like sensor inputs, but this seems like more of an advanced feature. 

If people do need OSC for interoperability there are [a couple of recommended solutions](http://integra.io/forum/discussion/47/does-integra-live-support-open-sound-control-osc) in the FAQ.


![screen shot 2015-11-10 at 09 26 14](https://cloud.githubusercontent.com/assets/579188/11058914/235182ca-878e-11e5-82b6-5fde7d8a8fd1.png)

**Comment by NiccoloGranieri** [comment]
@jamiebullock @elmiquito I totally agree with Jamie, and adding to the syncing issue with Ableton Live, last week they released a new "Link" button on the daw to sync immediately multiple Ableton timelines. We could probably look into that and use that to enhance integra!
